### PR TITLE
Enable to resize Events too

### PR DESCRIPTION
**Pull request by lucasarnaud**
I would like to resize BPMN Events and I found this plugin, but it only enable to resize tasks, activities and subprocesses.

Is there a reason so it is not possible currently?

I made this change to the code to enable to resize the Events too. I opened this PR if would like to add this feature to the original plugin.

Thanks for the plugin!

### CODE REVIEW

**Comment by ElCondor1969** [comment]
Hi @lucasarnaud .

It would be better to condition the resizing of events by a new option, for example _"eventResizingEnable"_, in such a way as to separate the two types of resizing and choose which one to enable independently of each other.

At this point it would be necessary to duplicate the _"ResizeTask.js"_ file in the new _"ResizeEvent.js"_ file in which we test the new _"eventResizingEnable"_ option and act as before but with the difference of testing whether the shape in question is an event , like from the code you added in the PR.

Finally, you should modify the "index.js" file to include the new _"ResizeEvent.js"_ file as well.

By doing this, the code is logically structured better.

**Comment by lucasarnaud** [comment]
Hi @ElCondor1969,

Thanks for the feedback. I've made the changes you suggested and they are already in the PR.

I added the new config to enable Event resizing (`eventResizingEnable`) to the README too.

File: README.md [hunk 1]
```diff
@@ -12,7 +12,23 @@ var bpmnJS = new BpmnModeler({
...
+In order to allow resize events to, set the `eventResizingEnabled` config:
+
+```javascript
+import BpmnModeler from 'bpmn-js/lib/Modeler';
+
+import resizeTask from 'bpmn-js-task-resize/lib';
+
+var bpmnJS = new BpmnModeler({
+  additionalModules: [
+    resizeTask
```
Comment (ElCondor1969) [inline README.md:28]: In your example, needs to add _resizeEvent_ module too in _additionalModules_ list.
Needs also to add the _import_ statement for the _resizeEvent_ module.

Comment (lucasarnaud) [reply README.md:28]: In my tests it worked this way, not modifying the import, because both modules are being exported in the same `index.js`

**Comment by ElCondor1969** [comment]
Well done. I'll merge the PR.

**Comment by lucasarnaud** [comment]
Hi @ElCondor1969, thanks for the merge!

When do you think you can make this changes available on NPM?

**Comment by ElCondor1969** [comment]
Hi @lucasarnaud.
I published to npm registry the **bpmn-js-task-resize@1.2.0** version.

**Pull request merged**